windows/src/engine/keyman32/k32_lowlevelkeyboardhook.cpp

```diff
@@ -304,31 +240,33 @@ LRESULT _kmnLowLevelKeyboardProc(
 }
 
 BOOL ProcessHotkey(UINT vkCode, BOOL isUp, DWORD ShiftState) {
-  if (UseRegisterHotkey()) {
-    return FALSE;
-  }
 
   Hotkeys *hotkeys = Hotkeys::Instance();   // I4641
   if (!hotkeys) {
+    SendDebugMessageFormat("Failed to get Instance");
     return FALSE;
   }
 
   Hotkey *hotkey = hotkeys->GetHotkey(ShiftState | vkCode);   // I4641
   if (!hotkey) {
+    SendDebugMessageFormat("GetHotkey Null");
     return FALSE;
   }
 
   if (isUp) {
+    SendDebugMessageFormat("Is Up");
```

Comment:
This doesn't seem like it should always be logged

Reply:
#12259.